### ISSUE TITLE
Can't send images in notify any more

### ISSUE THREAD

**Issue by sendorm** [home-assistant/home-assistant-iOS#158]
I am using the beta version of the ios app on ios 12. 
I can't seem to get any attachments in messages any more. 
This is the service call I use:
```
{
  "title": "Test plugin",
  "message": "Test",
  "data": {
    "attachment": {
        "url": "https://github.com/home-assistant/home-assistant-assets/blob/master/logo-round-192x192.png?raw=true","content-type":"png","hide-thumbnail": "false"
    }
  }
}
```

**Comment by mnoorenberghe** [comment]
Hmm… #140 covers authenticated URLs but your example is for a public URL…

**Comment by aaronwolen** [comment]
Ran into this same issue with the beta (currently 1.5.0-6) and confirmed re-installing the stable release (v1.1.1) fixed the issue.

**Comment by robbiet480** [comment]
Yep, this is a known issue at the moment, the camera and attachment code hasn't been updated to use the new API stuff quite yet.

**Comment by robbiet480** [comment]
Fixed via 28a27eb3592668dddfece963c065925bb2abba98
